packages/bot/src/assets/sass/scratch/toolbox.scss

```diff
@@ -77,7 +79,9 @@ $category-colours: (
         &__category-return {
             #{$toolbox}__category {
                 &--#{$category} {
-                    background-color: $colour;
+                    background-color: $lightgrey;
```

Comment:
Could we get these colours from `themes.scss`, or are they unique to bot? You may refer to `src/sass/app/_common/components/amount.scss` to see how it's being used in trader.

Reply:
you're probably right, but i just followed the existing code structure, perhaps i can do it within another card

Reply:
we need to refactor the way we handle colours in general, there's a card for in in the backlog